#26545 Fix: same .tsv file, get different data-frame structure using engine 'python' and 'c' (#26634)
pandas-dev/pandas: doc/source/whatsnew/v0.25.0.rst

- Bug in :func:`read_json` where date strings with ``Z`` were not converted to a UTC timezone (:issue:`26168`)
- Added ``cache_dates=True`` parameter to :meth:`read_csv`, which allows to cache unique dates when they are parsed (:issue:`25990`)
- :meth:`DataFrame.to_excel` now raises a ``ValueError`` when the caller's dimensions exceed the limitations of Excel (:issue:`26051`)

> - Fixed bug in :func:`pandas.read_csv` - when using engine='python', columns were handled incorrectly if the first header had in the bom (:issue:`26545`)

Reviewer: "if the first header had in the bom" ? - can you reword this?

Reviewer 2: ```suggestion
- Fixed bug in :func:`pandas.read_csv` - where a BOM would result in incorrect parsing using engine='python' (:issue:`26545`)
```
